src/container/home/home.js

```diff
@@ -0,0 +1,30 @@
+import Layout from '../../components/layout/layout';
+import Container from '../../components/container/container';
```

Comment:
Not a blocker, but something I wanted to note, that can help make your imports a little cleaner.

ie. In **src/components/container**, if you add an **indext.ts** file with following contents:

```
// src/components/container/index.ts
export * from './container';
```

Then import like this:

```
// src/components/home/home.js
import Container from '../../components/container';
```

We can take a step further by using **typescript aliases** (defined in tsconfig.json):

```suggestion
import Container from '@src/components/container';
```

I usually do this for all my imports because things can get pretty nested later.

Reply:
sounds great, i'll make this change after I merge the contact branch!